src/pages/my-courses/MyCourses.tsx

```diff
@@ -131,8 +139,35 @@ const MyCourses = () => {
     if (!duplicationError) await fetchData()
   }
 
+  const updateInfo = useCallback(() => {
+    void fetchData({
+      ...filters,
+      limit: itemsPerPage,
+      skip: (Number(filters.page) - 1) * itemsPerPage,
+      sort
+    })
+  }, [fetchData, filters, itemsPerPage, sort])
+
+  const searchString = searchParams.toString()
+
+  useEffect(() => {
+    updateInfo()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Do you mean a comment? Without it, eslint shows a warning "React Hook useEffect has a missing dependency: 'updateInfo'". But if I put this into dependencies, the page will be rerendered continuously.

Reply:
And why is this happening?

Reply:
To fix this only for my component I can convert `filters` object to string `filtersStr` and put it into dependencies.
But if we want to fix this also for other components, that use this `filters` (`FindOffers` for example), we must change `useFilterQuery` hook and make changes in each component, which depends on this hook. It would be better to do it in the new issue.

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/104922087/0b6ad9c8-9758-4fbe-a289-336c44802115)

So, what would you prefer: remove the comment and rewrite my component with converting object to string, or leave it untouched and create a new issue for rewriting `useFilterQuery` hook?

Reply:
Or another variant: add one more return variable in `useFilterQuery`:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/104922087/ddd26219-6067-4a89-9e68-b2bd92dd773e)

And then use it where it needed:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/104922087/53d8bc0f-a82c-4be3-8253-4f0112f8cff3)